clients/desktop/src/vault/deposit/DepositForm/index.tsx

```diff
@@ -171,6 +174,8 @@ export const DepositForm: FC<DepositFormProps> = ({
                     {t(
                       `chainFunctions.${selectedChainAction}.labels.${field.name}`
                     )}{' '}
+                    {field.name === 'amount' &&
+                      `(Balance: ${totalAmountAvailable.toFixed(2)} ${coin}) `}
```

Comment:
Hey @antoni0dev , can we please make it only show the balance for bond , NOT for unbond 

because `unbond` user might misunderstand it as the amount they will be able to unbond , instead of wallet balance